src/js/subway func.js

```diff
@@ -0,0 +1,48 @@
+import { Station } from './Station';
```

Comment:
These functions should belong to Parser why did you extract them to separate file?